commit 585ba4c17ad151d3afb671886f3a4eea883d5ea8 (refs/changes/18/4718/2)
Author: Robert Mustacchi <rm@joyent.com>
Date:   2018-08-17T23:31:23+00:00 (1 year, 2 months ago)
    
    OS-7033 Autogenerate Intel pcbe values from perfmon data (fix custr)
    Reviewed by: Jerry Jelinek <jerry.jelinek@joyent.com>
    Approved by: Jerry Jelinek <jerry.jelinek@joyent.com>

diff --git a/usr/src/tools/cpcgen/Makefile b/usr/src/tools/cpcgen/Makefile
index dc50c0b1e0..23f412439a 100644
--- a/usr/src/tools/cpcgen/Makefile
+++ b/usr/src/tools/cpcgen/Makefile
@@ -35,7 +35,7 @@ $(PROG): $(OBJS)
 %.o: $(SRC)/lib/json_nvlist/%.c
 	$(COMPILE.c) $<
 
-%.o: $(SRC)/lib/libcmdutils/common/%.c
+%.o: $(SRC)/lib/libcustr/common/%.c
 	$(COMPILE.c) $<
 
 clean:
